frontend/public/components/factory/_modal-wrapper.scss

```diff
@@ -6,6 +6,10 @@
   right: 0;
   top: 0;
   z-index: $zindex-modal;
+  .ReactModal__Body--open & {
```

Comment:
Nit:  insert blank line to be consistent within the file

Reply:
Maybe this should be more generic so we don't also need https://github.com/openshift/console/blob/master/frontend/public/components/catalog/_catalog.scss#L127-L128?

Reply:
Reworking this to enable similar scrolling behavior that's used in add to catalog.